run_linux.go

```diff
@@ -1069,6 +1071,7 @@ func setupRootlessNetwork(pid int) (teardown func(), err error) {
 
 func runConfigureNetwork(isolation define.Isolation, options RunOptions, configureNetworks []string, pid int, containerName string, command []string) (teardown func(), err error) {
 	var netconf, undo []*libcni.NetworkConfigList
+	logger := options.Logger
```

Comment:
This doesn't need to ever be different from `options.Logger`, and it's used once.  Remove the variable and reference `options.Logger` directly.

Reply:
That is the way I originally had it, and I thought this was cleaner, but I will revert.